cocos2d/core/renderer/RendererWebGL.js

```diff
@@ -603,8 +608,9 @@ return {
             gl.vertexAttribPointer(cc.VERTEX_ATTRIB_MVMAT0 + i, 4, gl.FLOAT, false, bytesPerRow * 4, matrixOffset + bytesPerRow * i); //stride is one row
         }
 
-        gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, _currentBuffer.buffer.elementBuffer);
-        gl.drawElements(gl.TRIANGLES, _currentBuffer.totalIndexSize, gl.UNSIGNED_SHORT, _currentBuffer.indexOffset);
+        var elemBuffer = getQuadIndexBuffer(count);
```

Comment:
Cool now you just using one index buffer everywhere, very clever! 
But my question is, how to change the rendering order without customizing index buffer? using z buffer?